snippets/csharp/System.Linq/Enumerable/AggregateTSource/enumerable.cs

```diff
@@ -803,10 +799,10 @@ static void ExceptEx1()
         static void FirstEx1()
         {
             // <Snippet35>
-            int[] numbers = { 9, 34, 65, 92, 87, 435, 3, 54,
-                                83, 23, 87, 435, 67, 12, 19 };
+            int[] numbers = [ 9, 34, 65, 92, 87, 435, 3, 54,
+                                83, 23, 87, 435, 67, 12, 19 ];
 
-            int first = numbers.First();
+            int first = numbers[0];

```

Comment:
Again, the purpose of this snippet is to demonstrate the `First` extension method. Changing to an index operator defeats the purpose of the sample.